sol/private/sol_binary.bzl

```diff
@@ -36,6 +36,10 @@ _ATTRS = {
         # allowed values can't be specified here
         default = ["abi", "bin", "hashes"],
     ),
+    "compiler_version": attr.string(),
```

Comment:
Please make this `solc_version`. It's a bit more concise and precise, and it matches with the `string_setting` name.

Reply:
okay sure, I looked at https://docs.soliditylang.org/en/develop/layout-of-source-files.html#version-pragma to see what name they use, as I generally try to make names line up with existing ecosystem documentation that users know about. but `solc_version` was on my shortlist as well